src/content/popup/tailwind.config.js

```diff
@@ -46,6 +50,14 @@ export default {
           'var(--base-font-size)',
           { lineHeight: 'calc(1.5 * var(--base-font-size))' },
         ],
+        kana: [
+          'calc(18 / 14 * var(--base-font-size))',
+          { lineHeight: 'calc(1.7 * var(--base-font-size))' },
+        ],
+        seireki: [
+          'calc(22 / 14 * var(--base-font-size))',
+          { lineHeight: 'calc(1.9 * var(--base-font-size))' },
+        ],
```

Comment:
I think you asked this in the names PR but, like I think you concluded there, we generally try to avoid semantic values here. `kanji` is a bit of a special case because it's so big and there are other elements that need to match its size (like the stroke animation SVG).

So I think for `kana`, 18 / 14 = 1.2857... which is pretty close to [Tailwind's `xl` size](https://github.com/tailwindlabs/tailwindcss/blob/cb518f462390ad09980964d01a52f7e0c0ae09bc/packages/tailwindcss/theme.css#L306) (1.25rem) so I think we should define an `xl` size with value `calc(18 / 14 * var(--base-font-size))` (and I guess a line height of 1.75).

For `seireki`, 22 / 14 = 1.5714... which is pretty close to [Tailwind's`2xl` size](https://github.com/tailwindlabs/tailwindcss/blob/cb518f462390ad09980964d01a52f7e0c0ae09bc/packages/tailwindcss/theme.css#L308) (1.5rem) so I think we should define a `2xl` size for that.

Reply:
> Ah, I see. I haven't thought about the logic behind this calculation until now. In the names tab, the kanji have a size of 20 / 14 = 1,43 which would then be between `xl` and `2xl`. Should we just define `2xl` as 21 / 14 = 1.5 and use it in both cases, here and in the names tab?
> 
> Edit: I'm doing so for now, but feel free to let me know if you want it to be 2 different sizes.

That sounds fine. I'll have a look. If we need two different sizes we can always define a `1.5xl` size.